array-destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,11 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+
+let GryfArray = hogwarts.filter((student) => {
+  return (student.house === "Gryffindor")
+})
+for (const student of GryfArray) {
+  let {firstName, lastName} = student;
+  console.log(firstName, lastName);
+}
```

Comment:
While this is a valid approach, could it be achieved with a single line of code?

This is a use case for the `forEach` higher-order function in JS.